featuretools/tests/wrappers/test_sklearn_wrapper.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from sklearn.preprocessing import StandardScaler
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import ExtraTreesClassifier
+from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import GridSearchCV
+import pickle
+
+from featuretools.demo.mock_customer import load_mock_customer
+from featuretools.wrappers.sklearn import DFSTransformer
+
+
+def test_sklearn_wrapper():
```

Comment:
can you split this method into multiple? each method should be named to describe what it does and only test one piece of functionality. 

if need look at how we use fixtures in other test files.